### DISCUSSION TITLE
[Webinar]Czwartki z Azure ➡️ Azure Kubernetes Services

### DISCUSSION THREAD

**Discussion by kaluzaaa** [AzureCommunityPL/group#50] [Community Content]
Dziś (03.09.2020) o 14:00 leci webinar, który prowadzę. Mocno podstawowy wprowadzający w Azure Kubernetes Services w ramach Czwartki z Azure.

- [Rejestracja tutaj](https://info.microsoft.com/CE-HCSAzureHybridInfra-WBNR-FY21-09Sep-03-CzwartkizAzureAzureKubernetesServices-SRDEM32059_LP01Registration-ForminBody.html)
- [Demo z webinaru](https://github.com/kaluzaaa/czwartek-z-azure-aks)
- [Slajdy](https://github.com/AzureCommunityPL/group/files/5166998/Czwartki.z.Azure.-.AKS.pdf)

**Comment by maciejgos** [comment]
Wiadomo może czy będzie i kiedy będzie nagranie dostępne na YouTube?

**Reply by kaluzaaa** [reply]
Będzie pewnie niedługo. Teraz można obejrzeć, jak się zarejestrujesz. Jest tam na platformie ciągle dostępne.
